### PR TITLE
[Enhancement] (nereids)implement showPluginsCommand in nereids

### LINKED ISSUE #42733: [Enhancement] (nereids)implement showPluginsCommand in nereids
showPlugins command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following showPlugins command in nereids:
unsupportedshowPluginsStatement
``` SHOW PLUGINS                                                                  ```

### PR DESCRIPTION
**Pull request by Vallishp**
### What problem does this PR solve?

implement show plugins in nereids

Issue Number: close #42733 

Related PR: #xxx

Problem Summary:

### Check List (For Committer)

- Test 

    - [ ] Regression test already exist.
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No colde files have been changed.
        - [ ] Other reason 

mysql> show plugins\G
*************************** 1. row ***************************
       Name: __builtin_AuditLoader
       Type: AUDIT
Description: builtin audit loader, to load audit log to internal table
    Version: 2.1.0
JavaVersion: 1.8.31
  ClassName: org.apache.doris.plugin.audit.AuditLoader
     SoName: NULL
    Sources: Builtin
     Status: INSTALLED
 Properties: {}
*************************** 2. row ***************************
       Name: __builtin_AuditLogBuilder
       Type: AUDIT
Description: builtin audit logger
    Version: 0.12.0
JavaVersion: 1.8.31
  ClassName: org.apache.doris.plugin.audit.AuditLogBuilder
     SoName: NULL
    Sources: Builtin
     Status: INSTALLED
 Properties: {}
*************************** 3. row ***************************
       Name: __builtin_SqlDialectConverter
       Type: DIALECT
Description: builtin sql dialect converter
    Version: 2.1.0
JavaVersion: 1.8.31
  ClassName: org.apache.doris.plugin.dialect.HttpDialectConverterPlugin
     SoName: NULL
    Sources: Builtin
     Status: INSTALLED
 Properties: {}
3 rows in set (3.17 sec)

- Behavior changed:

    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?

    - [ ] No.
    - [ ] Yes. 

- Release note

    
    
    None

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by Vallishp** [comment]
run buildall

**Comment by starocean999** [comment]
please resolve conflicts

**Comment by Vallishp** [comment]
run buildall

**Comment by Vallishp** [comment]
run buildall

**Comment by Vallishp** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowPluginsCommand.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+/**
+ * show plugins command
+ */
+public class ShowPluginsCommand extends ShowCommand {
+    public static final Logger LOG = LogManager.getLogger(ShowPluginsCommand.class);
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowPluginsCommand.java:42]:
```suggestion
```

**Review by LiBinfeng-01 — APPROVED** [review]

**Review by starocean999 — DISMISSED** [review]

**Comment by Vallishp** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Pull request merged**